modern/src/reports/DailySummaryReportPage.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { TableContainer, Table, TableRow, TableCell, TableHead, TableBody, Paper } from '@material-ui/core';
+import t from '../common/localization';
+import { formatDistance, formatHours, formatDate, formatSpeed, formatVolume } from '../common/formatter';
+import ReportFilter from './ReportFilter';
+import ReportLayoutPage from './ReportLayoutPage';
+import { useAttributePreference } from '../common/preferences';
+
+const ReportFilterForm = ({ onResult }) => {
+
+  const handleSubmit = async (deviceId, from, to) => {
+    const query = new URLSearchParams({
+      deviceId,
+      from: from.toISOString(),
+      to: to.toISOString(),
+      daily: true
+    });
+    const response = await fetch(`/api/reports/summary?${query.toString()}`, { headers: { Accept: 'application/json' } });
+    if (response.ok) {
+      onResult(await response.json());
+    }
+  }
+  return <ReportFilter handleSubmit={handleSubmit} />;
+}
+
+const DailySummaryReportPage = () => {
```

Comment:
I don't think we need a separate page for this. Maybe we can pass a parameter to the summary report?

Reply:
I thought that too. But created a separate page to sync with the report type Dropdown of the old web. Although now removed.